Core/Core/Configuration/Config/FeaturesConfig.swift

```diff
@@ -17,8 +17,8 @@ public class FeaturesConfig: NSObject {
     public var startupScreenEnabled: Bool
     
     init(dictionary: [String: Any]) {
-        whatNewEnabled = dictionary[FeaturesKeys.whatNewEnabled.rawValue] as? Bool ?? false
-        startupScreenEnabled = dictionary[FeaturesKeys.startupScreenEnabled.rawValue] as? Bool ?? false
+        whatNewEnabled = true//dictionary[FeaturesKeys.whatNewEnabled.rawValue] as? Bool ?? false
```

Comment:
@saeedbashir  are these debug values?

Reply:
Yes, it was pushed accidentally and reverted.